master/networking/vxlan-ipip.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Connecting workloads across networks that you do not control
+Description: Configure Calico to use IP in IP or VXLAN overlay networking so the underlying network doesn’t need to understand pod addresses.
```

Comment:
It's usually `IP-in-IP`

Reply:
Yes, I've seen that version but we settled on IP in IP in this doc that I did with Casey: https://docs.projectcalico.org/master/networking/vxlan-ipip. So I was trying to consistent with that.